core/templates/dev/head/css/oppia.css

```diff
@@ -111,7 +111,7 @@ body {
 }
 
 .oppia-profile-picture-fullsize {
-  height: 180px;
+  height: auto;
```

Comment:
Would you mind adding the following comment above this line:

    /* This maintains the aspect ratio of the picture. */

Otherwise, looks good to me -- thanks!